pkgs/extension_discovery/lib/extension_discovery.dart

```diff
@@ -100,17 +105,15 @@ final class Extension {
 ///
 /// ### Caching results
 ///
-/// When [useCache] is `true` then the detected extensions will be cached
-/// in `.dart_tool/extension_discovery/<targetPackage>.yaml`.
-/// This function will compare modification timestamps of
-/// `.dart_tool/package_config.json` with the cache file, before reusing cached
-/// results.
-/// This function will also treat relative path-dependencies as mutable
-/// packages, and check such packages for extensions every time [findExtensions]
-/// is called. Notably, it'll compare the modification time of the
-/// `extension/<targetPackage>/config.yaml` file, to ensure that it's older than
-/// the extension cache file. Otherwise, it'll reload the extension
-/// configuration.
+/// When [useCache] is `true` then the detected extensions will be cached in
+/// `.dart_tool/extension_discovery/<targetPackage>.yaml`. This function will
```

Comment:
```suggestion
/// `.dart_tool/extension_discovery/<targetPackage>.json`. This function will
```

Reply:
oops - my brain needs more coffee!

Reply:
Nevermind, this was existing code. Applied suggestion.